biogrid/apps/biogrid-mvp/src/app/pages/info-page/info-page.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import './info-page.css';
+import Visual from './EnergyFlowVisual.png';
+
+export const InfoPage = () => {
+
+
+  return (
+    <div className="info-page">
+      <h1>The Logic</h1>
+
+      <img src={Visual} id="EnergyFlow"></img>
+
+      <p>Firstly, the grid items <b>(batteries(b), solar panels(s), and buildings(b))</b> will be randomly placed in the town. 
```

Comment:
can you differentiate between batteries = b and buildings = b
maybe capitalize one or use ba for batteries and b for building?

Reply:
Awad refer to the DD for an update section for the version

Reply:
Thanks will do